CMakeLists.txt

```
@@ -359,26 +359,29 @@ if(${USE_MPI} MATCHES ON )
 endif(${USE_MPI} MATCHES ON )
 
 # If we do not locally compile ZLIB we will find ZLIB in the system, if we cannot find it we will enforce the local compilation
-if( LOCAL_ZLIB_COMPILATION MATCHES OFF )
+IF(NOT DEFINED FORCE_LOCAL_ZLIB_COMPILATION)
+    set( FORCE_LOCAL_ZLIB_COMPILATION OFF )
+ENDIF(NOT DEFINED FORCE_LOCAL_ZLIB_COMPILATION)
```

[check this out](https://cmake.org/cmake/help/v3.13/command/option.html):
```suggestion
OPTION ( FORCE_LOCAL_ZLIB_COMPILATION "force local compilation of zlib" OFF )
```

@roigcarlo any reason we don't use this more often?

---

Wjat does that black magic you are showing me?

---

I see, the link explains, hail to the link